samples/highcharts/series-arcdiagram/link-weight/demo.js

```diff
@@ -0,0 +1,64 @@
+Highcharts.chart('container', {
+
+    title: {
+        text: 'Highcharts Arc Diagram'
+    },
+    subtitle: {
+        text: 'Arc Diagram with fixed link weight'
+    },
+    series: [{
+        linkWeight: 1,
```

Comment:
I would add `centeredLinks: true,` to this demo:
![Zrzut ekranu 2022-01-12 o 11 00 11](https://user-images.githubusercontent.com/1453926/149118603-3594376a-928b-4f57-a762-5c8d45413c12.png)